WebDriverAgentLib/Utilities/FBScreenshot.m

```diff
@@ -64,14 +65,14 @@ + (NSString *)imageUtiWithQuality:(NSUInteger)quality
   }
 }
 
-+ (NSData *)takeWithQuality:(NSUInteger)quality
-                       rect:(CGRect)rect
-                      error:(NSError **)error
++ (NSData *)takeInOriginalResolutionWithQuality:(NSUInteger)quality
+                                           rect:(CGRect)rect
+                                          error:(NSError **)error
 {
   if ([self.class isNewScreenshotAPISupported]) {
     XCUIScreen *mainScreen = XCUIScreen.mainScreen;
     return [self.class takeWithScreenID:mainScreen.displayID
-                                  scale:mainScreen.scale
+                                  scale:SCREENSHOT_SCALE
                      compressionQuality:[self.class compressionQualityWithQuality:FBConfiguration.screenshotQuality]
                                    rect:rect
```

Comment:
We should also update the documentation for `rect` argument in `takeWithScreenID` API as well as in `scaledImageWithImage`.
Also, it makes sense to do a check for the scale factor 1.0 in `scaledImageWithImage` API, so we don't do unnecessary drawing there